CHANGELOG.md

```diff
@@ -47,6 +47,11 @@ Thank you to all who have contributed!
 - **Breaking**: Modifies `functions` property of `CompilerPipeline`, `StaticTypeInferencer` and  `StepContext` to be a `List` and not a `Map`.
 
 ### Deprecated
+- Incorporates `@Deprecated` annotations with a `WARNING` level to the `CompilerPipeline` 
+  and experimental `PartiQLCompilerPipeline`, providing an alternative API for use 
+  in place of the deprecated methods. 
+- Integrates `@Deprecated` annotations with an `ERROR` level to the `ExprFunction` and 
+  `FunctionSignature`, throwing errors when the deprecated methods are invoked.
```

Comment:
I'd reword these for each deprecated class/method such as:
```
- Deprecated XXX. Please use XX instead.
- **Breaking**: Deprecated XX with a Deprecation Level of ERROR. Please use XX instead.
```

Note, that some of your previous CHANGELOG statements may conflict with this -- so make sure to update.